.github/workflows/windows.yml

```diff
@@ -11,8 +11,14 @@ jobs:
   windows-build-gui:
     strategy:
       matrix:
-        qtversion: [5.12.8, 5.14.2]
-    runs-on: windows-2016
+        qtversion: [5.15.2]
+        include:
+          - qtversion: 5.15.2
+            qtarch: win64_msvc2019_64
+            cmake-generator: "Visual Studio 17 2022"
```

Comment:
I think this will require updating the Boost dependency to something newer, and possibly python as well. It might break a lot of code. I have SCIRun building with Boost 1.75, but that version is also too old for VS 2022.